lib/src/ffmpeg/filters/audio_mix_filter.dart

```diff
@@ -2,14 +2,15 @@ import 'package:ffmpeg_cli/src/ffmpeg/ffmpeg_command.dart';
 
 /// Mixes multiple audio streams together into one.
 ///
-/// I think this will automatically cut the volume of each input proportional
+/// This will automatically cut the volume of each input proportional
 /// to the number of inputs, e.g., 1/2 volume for 2 inputs, 1/3 volume for 3
-/// inputs.
+/// inputs. (1/n)
```

Comment:
I don't know why the "(1/n)" was added to the end of this line.

Reply:
Removed as it is unnecessary - I was just adding a notation so the user knows how to calculate volume but don't see the point in it now.